router/core/router.go

```diff
@@ -135,7 +135,11 @@ type (
 		rootContext       context.Context
 		rootContextCancel func()
 		routerConfig      *nodev1.RouterConfig
-		healthChecks      health.Checker
```

Comment:
There's no reason for the healthChecks to be on the currently active server vs the main router. I pulled it up to the main Router instance.